scripts/diff-coverage.py

```
@@ -0,0 +1,507 @@
+        if line.startswith("--- "):
+            continue
+        m = _HUNK_RE.match(line)
+        if m:
+            if current is None:
+                raise ToolError(f"hunk 出现在未识别文件头之后: {line!r}")
+            new_lineno = int(m.group(1))
+            in_hunk = True
+            continue
+        if not in_hunk or current is None:
+            continue
+        if line.startswith("+"):  # 新增行（含修改行的新侧）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)